pkg/iac-providers/kubernetes/v1/load-file.go

```diff
@@ -0,0 +1,48 @@
+package k8sv1
+
+import (
+	"github.com/accurics/terrascan/pkg/utils"
+
+	"github.com/accurics/terrascan/pkg/iac-providers/output"
+	"go.uber.org/zap"
+)
+
+// LoadIacFile loads the k8s file specified
+// Note that a single k8s yaml file may contain multiple resource definitions
+func (k *K8sV1) LoadIacFile(absRootPath string) (allResourcesConfig output.AllResourceConfigs, err error) {
+	allResourcesConfig = make(map[string][]output.ResourceConfig)
+
+	var iacDocuments []*utils.IacDocument
+
+	fileExt := k.getFileType(absRootPath)
+	switch fileExt {
+	case YAMLExtension:
+		fallthrough
+	case YAMLExtension2:
+		iacDocuments, err = utils.LoadYAML(absRootPath)
+	case JSONExtension:
+		iacDocuments, err = utils.LoadJSON(absRootPath)
+	default:
+		zap.S().Warn("unknown extension found", zap.String("extension", fileExt))
```

Comment:
agreed--should be an error. The default case would normally only be hit if specifying the -f param directly, and since we support both yaml/json formatted k8s files, no ext is ambiguous